docs/examples/styled_components.md

```diff
@@ -0,0 +1,5 @@
+# styled-components
```

Comment:
@neilhsmith +1 I was expecting an exemple with styled-components for a long time. Thanks!